dolphin/api/v1/alert.py

```diff
@@ -112,6 +124,14 @@ def _input_check(self, alert_source):
                         raise exception.InvalidInput(reason=msg)
                     alert_source['privacy_key'] = cryptor.encode(
                         alert_source['privacy_key'])
+                else:
```

Comment:
multiple else an be removed.
ex:    
add this top of if block and remove all else  
                alert_source['auth_key'] = None
                alert_source['auth_protocol'] = None
                alert_source['privacy_key'] = None
                alert_source['privacy_protocol'] = None

Reply:
Thanks, done !